lib/pavilion/plugins/commands/show.py

```diff
@@ -771,6 +783,40 @@ def _tests_cmd(self, pav_cfg, args):
             title="Available Tests"
         )
 
+
+    @show_cmd('series')
+    def _series_cmd(self, pav_cfg, args):
+
```

Comment:
See the resolver's `find_all_tests()` method. You may want to move this functionality to somewhere else to provide it more generally.